packages/copyable/src/Copyable.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ClipboardJS from 'clipboard';
+import { transparentize } from 'polished';
+
+import Button, { Variant as ButtonVariant } from '@leafygreen-ui/button';
+import { css, cx } from '@leafygreen-ui/emotion';
+import CopyIcon from '@leafygreen-ui/icon/dist/Copy';
```

Comment:
Do we think it'd be a good idea to make a lint rule to enforce these imports to reduce the chance of regression?

I think https://eslint.org/docs/rules/no-restricted-imports would make it easy.

Reply:
what restriction(s) would you have in mind?